wled00/bus_manager.h

```diff
@@ -344,7 +392,7 @@ struct BusConfig {
   {
     refreshReq = (bool) GET_BIT(busType,7);
     type = busType & 0x7F;  // bit 7 may be/is hacked to include refresh info (1=refresh in off state, 0=no refresh)
-    size_t nPins = Bus::getNumberOfPins(type);
+    size_t nPins = OUTPUT_MAX_PINS;
```

Comment:
We are using the "pins" in the config file for more than just GPIO, so always copy the full array

Reply:
I have checked the places the array is created and replaced the magic number 5 with the proper constant

Reply:
If I don't change, then we *would* see an out of bounds as the number of GPIO pins used is 16 but there are only 3 "pins" used in the config